2022-11-29T20:56:02.728436989Z stderr F I1129 20:56:02.728341       1 node.go:163] Successfully retrieved node IP: 172.18.0.3
2022-11-29T20:56:02.728454114Z stderr F I1129 20:56:02.728381       1 server_others.go:138] "Detected node IP" address="172.18.0.3"
2022-11-29T20:56:02.728632572Z stderr F I1129 20:56:02.728592       1 server_others.go:578] "Unknown proxy mode, assuming iptables proxy" proxyMode=""
2022-11-29T20:56:02.755575364Z stderr F I1129 20:56:02.753122       1 server_others.go:206] "Using iptables Proxier"
2022-11-29T20:56:02.755593572Z stderr F I1129 20:56:02.753139       1 server_others.go:213] "kube-proxy running in dual-stack mode" ipFamily=IPv4
2022-11-29T20:56:02.755595572Z stderr F I1129 20:56:02.753143       1 server_others.go:214] "Creating dualStackProxier for iptables"
2022-11-29T20:56:02.755598114Z stderr F I1129 20:56:02.753151       1 server_others.go:501] "Detect-local-mode set to ClusterCIDR, but no IPv6 cluster CIDR defined, , defaulting to no-op detect-local for IPv6"
2022-11-29T20:56:02.755599822Z stderr F I1129 20:56:02.753222       1 proxier.go:259] "Setting route_localnet=1, use nodePortAddresses to filter loopback addresses for NodePorts to skip it https://issues.k8s.io/90259"
2022-11-29T20:56:02.755601572Z stderr F I1129 20:56:02.753279       1 proxier.go:259] "Setting route_localnet=1, use nodePortAddresses to filter loopback addresses for NodePorts to skip it https://issues.k8s.io/90259"
2022-11-29T20:56:02.755602822Z stderr F I1129 20:56:02.753494       1 server.go:661] "Version info" version="v1.24.6"
2022-11-29T20:56:02.755603989Z stderr F I1129 20:56:02.753498       1 server.go:663] "Golang settings" GOGC="" GOMAXPROCS="" GOTRACEBACK=""
2022-11-29T20:56:02.755605197Z stderr F I1129 20:56:02.753874       1 conntrack.go:52] "Setting nf_conntrack_max" nf_conntrack_max=131072
2022-11-29T20:56:02.755606947Z stderr F I1129 20:56:02.753917       1 conntrack.go:100] "Set sysctl" entry="net/netfilter/nf_conntrack_tcp_timeout_close_wait" value=3600
2022-11-29T20:56:02.755608072Z stderr F I1129 20:56:02.754319       1 config.go:317] "Starting service config controller"
2022-11-29T20:56:02.755609197Z stderr F I1129 20:56:02.754352       1 shared_informer.go:255] Waiting for caches to sync for service config
2022-11-29T20:56:02.755610322Z stderr F I1129 20:56:02.754362       1 config.go:226] "Starting endpoint slice config controller"
2022-11-29T20:56:02.755611489Z stderr F I1129 20:56:02.754363       1 shared_informer.go:255] Waiting for caches to sync for endpoint slice config
2022-11-29T20:56:02.755612614Z stderr F I1129 20:56:02.755228       1 config.go:444] "Starting node config controller"
2022-11-29T20:56:02.755613864Z stderr F I1129 20:56:02.755235       1 shared_informer.go:255] Waiting for caches to sync for node config
2022-11-29T20:56:02.854850156Z stderr F I1129 20:56:02.854738       1 shared_informer.go:262] Caches are synced for endpoint slice config
2022-11-29T20:56:02.854885156Z stderr F I1129 20:56:02.854771       1 shared_informer.go:262] Caches are synced for service config
2022-11-29T20:56:02.856032864Z stderr F I1129 20:56:02.855989       1 shared_informer.go:262] Caches are synced for node config
2022-11-29T21:00:03.803891979Z stderr F E1129 21:00:03.803781       1 service_health.go:187] "Healthcheck closed" err="accept tcp [::]:31763: use of closed network connection" service="default/keycloak-db"
